### ISSUE TITLE
Decide on first chapters draft

### ISSUE THREAD

**Issue by Globegitter** [sane/sane-book#2]
https://github.com/sane/sane-book/blob/master/SUMMARY.md has @mgenev proposed chapter format. This is now open to discussion to add any missing things or shuffle them around a bit if other people have suggestions.

Just comment below with any ideas.

**Comment by Globegitter** [comment]
I really like the proposed structure, maybe once I would have one or two more ideas for some advanced topics as I am working on my projects but :+1: otherwise.

We just have to make sure that this follows coherent storyline so it makes it easy for people to follow.

**Comment by jamemackson** [comment]
I also like the structure that @mgenev proposed.  I'm sure as we get into it further that I'll have more ideas to tweak it but think it will serve us well as a starting point.  I also agree with @Globegitter that we should really make sure that the storyline flows.  

And to that note, thinking aloud here, i think Part I is an obvious (and good) way to begin.  Part II looks to me like it would correlate to learning the fundamentals and having a basic app structure w/ authentication in place when the reader completes that section.  Part III then seems like a logical extension to II where we would add in user roles to the basic authentication and add more functionality like something real time with sockets and mapping something w/ geodata etc.  and then start addressing things like testing deployment and CI.  

So yea, I think we should be able to make a logical flow for the content based on these rough ideas.  Very exciting stuff everyone!  @mgenev thanks for stepping right in and getting all of this carved out so far!

**Comment by jhliberty** [comment]
I like the proposed structure as well - It's atleast a nice base to work from.. As we move through and start building the content we may end up shifting something around or changing a name here or there but it's solid.

I created a new branch to start adding these page and setting up the structure in the summary.

**Comment by mgenev** [comment]
I agree that more advanced topics are needed. I may have some more in my notes. I was also thinking adding an appendix of various features analyzed that aren't necessarily part of the chapter sequence
